.github/workflows/main.yml

```diff
@@ -77,3 +78,75 @@ jobs:
           npm run test-install
           # Undo those changes.
           git checkout --force
+
+  macos:
+    # https://github.com/actions/virtual-environments#available-environments
+    runs-on: macos-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
+          ref: ${{ github.head_ref }}
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v1
+        with:
+          # Test only the oldest maintenance LTS Node.js version.
+          # https://github.com/nodejs/Release#release-schedule
+          node-version: 10
+
+      - name: Install dependencies
+        run: |
+          # Test platform-specific browser binary fetching for both
+          # Chromium and Firefox.
+          PUPPETEER_PRODUCT=firefox npm install
```

Comment:
Either add `Run unit tests on Firefox` or don't install Firefox. Same for Windows.

Reply:
For macOS and Windows, we don't run exhaustive checks on purpose: testing everything on all platforms would take too much time. This line tests that the platform-specific binary fetching works properly on macOS (see the comment above), which is useful independently of the unit tests.

We intentionally don't run Firefox unit tests _on all platforms_ (instead, we only do so on Linux, as part of our exhaustive checks), but we do check whether `npm install` (which downloads the platform-specific binary) works.